doc/developer_guide/development.rst

```diff
@@ -1,48 +1,45 @@
 🚧 Development - Contributing
 =============================
 
-Setting up the Development Environment
-++++++++++++++++++++++++++++++++++++++
+Creating a Release
+++++++++++++++++++
 
-.. figure:: ../_static/nothing-to-see-here.png
-    :alt: nothing to see here
-    :target: https://imgflip.com/i/2a7gqa
+Prepare the Release
+-------------------
 
-    source: `imgflip.com <https://imgflip.com/>`_
+To prepare for a release, a pull request with the following things needs to be created:
 
-Creating a Release
-++++++++++++++++++
+- Updated version numbers
+- Updated the changelog
+- Updated workflow templates (not automated yet)
+
+This can be achieved by running the following command:
 
-Prerequisites
--------------
+.. code-block:: shell
 
-* Change log needs to be up to date
-* Latest change log version needs to match project and package version
-* Release tag needs to match package, changelog and project version
+   nox -s prepare-release -- <major>.<minor>.<patch>
```

Comment:
Would it be possible to use the current project version as default?

Reply:
Is there a check to avoid skipping intermediary release numbers? 
E.g. jumping from 0.1.0 to 1.0.0 might be OK, while 0.1.0 to 0.5.0 is not.

Reply:
I don't think skipping release numbers can be avoided in a SemVer context.

E.g.: `0.1.0`, `0.2.0` - was `0.1.1` skipped?

From my point of view, the version number in a SemVer-based project must always be deliberately chosen. Therefore, I intentionally force the user to pick it. That said, it is possible to supply the currently set one (if you are just doing a release, based on existing changes) `version >= current`, but you still need to actively decide if that version still makes sense.